Packs/AWS-CloudTrail/Integrations/AWS-CloudTrail/AWS-CloudTrail.py

```diff
@@ -230,15 +277,47 @@ def describe_trails(args: dict) -> CommandResults:
         output.append(trail)
 
     raw = json.loads(json.dumps(output))
-    return CommandResults(
-        outputs_prefix="AWS.CloudTrail.Trails",
-        outputs_key_field="Name",
-        outputs=raw,
-        readable_output=tableToMarkdown('AWS CloudTrail Trails', data),
+    ec = {'AWS.CloudTrail.Trails(val.Name == obj.Name)': raw}
+    human_readable = tableToMarkdown('AWS CloudTrail Trails', data)
+    return_outputs(human_readable, ec)
+
+
+def get_trail_status(args):
+    client = aws_session(
+        region=args.get('region'),
+        roleArn=args.get('roleArn'),
+        roleSessionName=args.get('roleSessionName'),
+        roleSessionDuration=args.get('roleSessionDuration'),
     )
 
+    kwargs = {'Name': args.get('name')}
+
+    try:
```

Comment:
No need to put under try &. except, we use try & except on the main function to catch all issues.